__tests__/app.test.js

```diff
@@ -61,3 +63,40 @@ describe("/api", () => {
       });
   });
 });
+
+//QUESTION 4
+describe("/api/articles/:article_id", () => {
+  test("GET: 200 sends a single article", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article.article_id).toBe(1);
+        expect(body.article.author).toEqual(articleData[0].author);
```

Comment:
you could use a forEach to loop through so you wouldnt need [0] on each of these